Sources/Kitura/SwaggerGenerator.swift

```diff
@@ -316,11 +316,21 @@ struct SwaggerDocument: Encodable {
     // - Returns: SwaggerParameter.
     func buildParameter(name: String, parametertype: String) -> SwaggerParameter {
         if name == "id" {
+            // id can only be string or integer. so force the type to string if
+            // it is neither.
+            var idType = "string"
+            if let ptype = SwiftType(rawValue: parametertype) {
```

Comment:
I think I understand what's going on here, but it would be nice to make a few improvements:

1. `parametertype` -> `parameterType`
2. `swaggerType()` should be a property not a function
3. If the name can only be String or Integer why not define that in an enum rather than passing Strings everywhere